### PR TITLE
add symbols, update local config

### PR DESCRIPTION
**Pull request by gitcontributor**
## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**

#120 

**What is the new behavior?**

 Symbols default .svg items is not missing

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: product/assets/symbols/map-pin-marked.svg [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (MV88) [inline product/assets/symbols/map-pin-marked.svg:1]: - remove product folder 
- move the symbols folder inside  root_of_project/assets

root_of_project is where localConfig is placed

**Comment by gitcontributor** [comment]
@MV88  I checked the symbol styles change on my local machine and it works fine.

File: product/assets/symbols/map-pin-marked.svg (hunk 1 above)
Comment (gitcontributor) [reply product/assets/symbols/map-pin-marked.svg]: done

File: localConfig.json [hunk 2]
```diff
@@ -330,6 +330,12 @@
           }
         }
       }
+    },
+    {
+      "name": "Annotations",
```
[the remark this answers is not in the document]
Comment (gitcontributor) [reply localConfig.json]: done

**Pull request merged**